### DISCUSSION TITLE
How to call phase parameter in submodels that are not dorectly related to phase?

### DISCUSSION THREAD

**Discussion by RuiheLi** [pybamm-team/PyBaMM#2421] [Q&A] [answered]
Dear  PyBaMM,

I find that in the submodels files, like base_sei.py, when we want to call phase parameters, we do 

`a = self.phase_param.a`

However, what if we want to call phase parameters in other submodel like electrolyte diffusion? If we do the same, it will say something like 

`xxx object has no attribute 'phase_param'`

Thanks so much!

**Comment by valentinsulzer** [comment, marked as answer]
There's no concept of phase for the electrolyte submodel. Which parameter are you trying to access?

**Reply by RuiheLi** [reply]
Thanks for the kind reply!

Quite a few, for example, c_ec_0_dim, and some others which are used in the SEI sub-model.

**Reply by valentinsulzer** [reply]
You can do `param.n.prim.c_ec_0_dim` for the EC concentration of the primary phase, `param.n.sec.c_ec_0_dim` for the secondary phase
